packages/utils/scaffold.ts

```diff
@@ -11,6 +11,29 @@ import astTransform from "./recursive-parser";
 import runPrettier from "./run-prettier";
 import { Node } from "./types/NodePath";
 
+
+
+function mergeHandler(config: Config, transformations: string[]): [Config, string[]]{
+	if(!config["topScope"])
```

Comment:
Assuming you already declared the array in the constructor, you won't need to check if it's existent, you only need to push to the array

Reply:
Added to avoid errors if runTransform function is directly used, like I do in webpack UI

Reply:
Will found some other way around for UI 👍 . Removed from here.